### DISCUSSION TITLE
cron environment issue (dev, qa, prod)

### DISCUSSION THREAD

**Discussion by glappen** [dokku/dokku#5024] [Q&A] [answered]
Hello, I just started using app.json to define my cron jobs today, but I have a question. Is there any way to disable the cron jobs in the lower environments like our qa and uat environments? I only want the jobs to run on production, but same codebase gets deployed to different places. Thanks!

**Comment by josegonzalez** [comment]
No, there is currently no way to disable cron tasks. Might be a nice feature though! Feel free to file a feature request.

**Comment by zodman** [comment, marked as answer]
the best wey its set.

app.prod.json 
app.uat.json

and set inside a custom formation https://dokku.com/docs/processes/process-management/#manually-managing-process-scaling

**Comment by AThilenius** [comment]
@zodman You're awesome, thank you!

Just to clarify (because it took me a min to figure out... I'm slow) create an `app.prod.json` file et. al, then set it on the app deployment for production via `dokku app-json:set app-name appjson-path app.prod.json`. This is a perfect solution to the OP's question about CRON for prod only.
